app/schemas/models/level_session.coffee

```diff
@@ -37,8 +37,6 @@ _.extend LevelSessionSchema.properties,
     type: 'string'
   levelID:
     type: 'string'
-  multiplayer:
```

Comment:
I already unset almost all of them. There may be some code live that continues to set it, so we could pre-deploy the removal of that, remove the flags, and then deploy live, or we could just deploy live and at the same time run this script (shouldn't take long now):

```
db.level.sessions.update({multiplayer: {$exists: true}}, {$unset: {multiplayer: 1}}, {multi: true})
```